### PR TITLE
fix(hybridcloud) Add index to apitoken.token

### PR DESCRIPTION
**Pull request by markstory**
This column is used in many queries and an index should help improve query performance.

### CODE REVIEW

**Review by dashed — APPROVED** [review]

File: src/sentry/models/apitoken.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ class ApiToken(ReplicatedControlModel, HasApiScopes):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_apitoken"
+        indexes = (models.Index(fields=["token"]),)
```
Comment (wedamija) [inline src/sentry/models/apitoken.py:56]: This is already unique, are you sure there's not already an index here?

Comment (wedamija) [reply src/sentry/models/apitoken.py:56]: I can see the unique index in prod - `CREATE UNIQUE INDEX sentry_apitoken_pkey ON public.sentry_apitoken USING btree (id)`

Comment (wedamija) [reply src/sentry/models/apitoken.py:56]: Should this be added here instead? https://github.com/getsentry/sentry/blob/65df63f82a417585cebbd4bd640cebc6daabb774/src/sentry/hybridcloud/models/apitokenreplica.py#L20

Comment (markstory) [reply src/sentry/models/apitoken.py:56]: > Should this be added here instead? 

I have a separate pull request for the replicamodel. I totally missed the unique attribute on the field, and only noticed the absence of any `indexes`.

 I'll close this as it isn't going to improve query plans.

**Pull request closed without merging**